docs/satellite-configuration-tuning.rst

```diff
@@ -70,6 +70,8 @@ qdrouterd and qpid tunings
 Calculating the maximum open files limit for qdrouterd
 ------------------------------------------------------
 
+In deployments using katello-agent infrastructure with a large number of Content Hosts, it may be necessary to increase the maximum open files for qdrouterd.
```

Comment:
This may not be as obvious that it is optional to users starting with 6.10+, do you think it would be worth while to mention:

 a) katello-agent infrastructure is optional starting with 6.10+
 b) it can be disabled if not used with 6.10+ which can increase performance by freeing up resources

Reply:
What about adding extra section about this with steps on how to disable it? @Imaanpreet what do you think?

Reply:
Yes, we can mention these steps.